### PR TITLE
Homework 3 completed

### PR DESCRIPTION
**Pull request by nechepurenkoN**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -82,6 +82,24 @@
...
+      <artifactId>webdrivermanager</artifactId>
+      <version>${webdrivermanager.version}</version>
+      <scope>compile</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.projectlombok</groupId>
+      <artifactId>lombok</artifactId>
+      <version>${lombok.version}</version>
+      <scope>compile</scope>
+    </dependency>
```
Comment (shoggy) [inline pom.xml]: Duplicated dependencies

File: src/main/java/utils/JdiTestingIndexPageData.java [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+    public static final String PAGE_URL = BASE_URL + "index.html";
+
+    public static final String PAGE_TITLE = "Home Page";
+
+    public static final String LOGIN = getLogin();
+
+    public static final String PASSWORD = getPassword();
+
+    public static final String USERNAME = "Roman Iovlev";
```
Comment (shoggy) [inline src/main/java/utils/JdiTestingIndexPageData.java]: good candidate for externalization, like login/password

**Review by shoggy — CHANGES_REQUESTED** [review]
Please fix Compilation Errors

**Comment by nechepurenkoN** [comment]
@shoggy I changed scope of those libs (`testng`, `lombok`, `webdrivermanager`) to default (`compile`) in order to use them in `src` files. I fixed compilation problems after I replaced a file from `hw2`,  now `hw2` package is in a correct state and project compiles without errors.

Also I add User pojo to encapsulate user credentials.

**Review by shoggy — APPROVED** [review]

**Pull request merged**